include/featurelib.h

```diff
@@ -0,0 +1,18 @@
+#ifndef FEATUREDETECT_FEATURELIB_H_
+#define FEATUREDETECT_FEATURELIB_H_
+
+#include <vector>
+#include <iostream>
+#include <opencv2/opencv.hpp>
+#include <opencv2/core/core.hpp>
+#include <opencv2/features2d.hpp>
+#include <opencv2/imgproc.hpp>
+
+class featurelib  {
```

Comment:
There is an extra space between `featurelib` and `{` -- CLion should fix such issues when you press `Ctrl + Alt + L`

Reply:
done